package.json

```diff
@@ -5,14 +5,17 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./src/bin/www",
-    "devstart": "babel-watch ./src/index.js",
+    "devstart": "babel-watch --exec babel-node ./src/index.js",
     "build": "rm -rf dist && mkdir dist && babel src -s -d dist",
-    "test": " set NODE_ENV=test&& mocha -r @babel/register --recursive src/tests/ --exit",
+    "test": " set NODE_ENV=test && npm run reset:db && mocha -r @babel/register --recursive src/tests/ --exit --timeout 10000",
     "generate-lcov": "nyc report --reporter=text-lcov > lcov.info",
```

Comment:
why is this here?
coveralls alone is okay

Reply:
I don't understand what you're asking, I didn't touch the coveralls

Reply:
i mean `generate-lcov`

Reply:
I also forgot this: You need to create two databases, one for `development` and one for `test`. During tests we set database_url to that of `test` and at the end of all tests drop all its tables.

Reply:
- ok I see, but I'm not the one that added the geerated l-cov.
- For the test databse url there is already provision for that in the sequelize config file, it just needs a little adjustment, that may be out of the scope of this branch.